src/main/drivers/io.h

```diff
@@ -98,6 +98,14 @@
 # warning "Unknown TARGET"
 #endif
 
+#if defined(STM32F7)
+// Expose these for EXTIConfig
+#define IOConfigGetMode(cfg) (((cfg) >> 0) & 0x03)
```

Comment:
It is better to keep macros UPPER_CASE. Or `static inline` functions may be used ...

Reply:
Done.